### DISCUSSION TITLE
More colors in NamedTextColor

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1681] [Help] [answered]
Hey, I want to use more colors than that are existing in the NamedTextColor class.

Im making many teams, and I want that players can decide their team color.

What is the best way to make it possible to use more colors?

**Comment by Tofaa2** [comment]
MiniMessage API can be used to convert hex codes into colors. https://docs.adventure.kyori.net/minimessage/index.html

**Reply by DevBlokJochem** [reply]
Hey, this usage of the api is great for messages, but I want to set the teamcolor with custom colors.

**Reply by GoldenCookieGIT** [reply]
iirc team colors are hard coded to those default colors

**Reply by DevBlokJochem** [reply]
ah, that's sad

**Comment by GoldenCookieGIT** [comment, marked as answer]
If you're trying to change glow color, you're locked to those colors. You could set the prefix to a custom color if you want the name tags to be different
